oneko.js

```diff
@@ -196,6 +196,50 @@
     idleAnimationFrame += 1;
   }
 
+  function explodeHearts() {
+    console.log('explodeHearts');
```

Comment:
Is this console.log still supposed to be there?

Reply:
No, it's not. That's some left over code from my debugging, earlier. I'll fix this as well as well as another section that was missing in one moment